packages/plans-grid-next/src/hooks/data-store/use-plan-for-spotlight.ts

```diff
@@ -0,0 +1,31 @@
+import { getPlanClass } from '@automattic/calypso-products';
+import { useMemo } from 'react';
+import { GridPlan } from '../../types';
+
+const SPOTLIGHT_ENABLED_INTENTS = [ 'plans-default-wpcom' ];
```

Comment:
This context can and should live outside IMO. It's also part of the condition for showing the spotlight, so best to centralize behind `isSpotlightOnCurrentPlan` from `plans-features-main`.

Going forward, let's always be critical about use of "intent" or "flowname" variables in the grid.